code/turf/turf.dm

```diff
@@ -2,57 +2,57 @@
 	icon = 'icons/turf/floors.dmi'
 	plane = PLANE_FLOOR //See _plane.dm, required for shadow effect
 	appearance_flags = TILE_BOUND | PIXEL_SCALE
-	var/intact = 1
-	var/allows_vehicles = 1
+	var/intact = TRUE
+	var/allows_vehicles = TRUE
 
-	var/tagged = 0 // Gang wars thing
+	/// Gang wars thing
+	var/tagged = 0
 
 	level = 1
 
 	unsimulated
 		pass_unstable = FALSE
-		var/can_replace_with_stuff = 0	//If ReplaceWith() actually does a thing or not.
+		/// If ReplaceWith() actually does a thing or not.
+		var/can_replace_with_stuff = FALSE
 #ifdef RUNTIME_CHECKING
-		can_replace_with_stuff = 1  //Shitty dumb hack bullshit
+		can_replace_with_stuff = TRUE  //Shitty dumb hack bullshit
 #endif
-		allows_vehicles = 0
+		allows_vehicles = FALSE
 
 	proc/burn_down()
 		return
 
-		//Properties for open tiles (/floor)
+	/// Properties for open tiles (/floor)
```

Comment:
```suggestion
	// Properties for open tiles (/floor)
```
I think this expands into multiple properties, and the above doc comment would only apply to the first one